docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+phpfpm:
+  command: php-fpm --allow-to-run-as-root
+  image: php:5.6.8-fpm
+  volumes:
+    - fpm.conf:/usr/local/etc/php-fpm.conf
```

Comment:
@alexanderilyin FYI if I mount a directory, instead of a file, then it does build.